src/patternfly/components/Nav/nav.scss

```diff
@@ -391,119 +475,6 @@
-  }
-
-  .pf-c-nav__link:active {
-    --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__horizontal-list__link--active--before--BorderWidth);
-  }
-
-  .pf-c-nav__link.pf-m-current {
-    --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__horizontal-list__link--m-current--before--BorderWidth);
-  }
-}
-
-// Tertiary list
-.pf-c-nav__tertiary-list {
-  --pf-c-nav__link--PaddingTop: var(--pf-c-nav__tertiary-list__link--PaddingTop);
-  --pf-c-nav__link--PaddingRight: var(--pf-c-nav__tertiary-list__link--PaddingRight);
-  --pf-c-nav__link--PaddingBottom: var(--pf-c-nav__tertiary-list__link--PaddingBottom);
-  --pf-c-nav__link--PaddingLeft: var(--pf-c-nav__tertiary-list__link--PaddingLeft);
-  --pf-c-nav__link--FontWeight: var(--pf-c-nav__tertiary-list__link--FontWeight);
-  --pf-c-nav__link--Color: var(--pf-c-nav__tertiary-list__link--Color);
-  --pf-c-nav__link--hover--Color: var(--pf-c-nav__tertiary-list__link--hover--Color);
-  --pf-c-nav__link--active--Color: var(--pf-c-nav__tertiary-list__link--active--Color);
-  --pf-c-nav__link--focus--Color: var(--pf-c-nav__tertiary-list__link--focus--Color);
-  --pf-c-nav__link--m-current--Color: var(--pf-c-nav__tertiary-list__link--m-current--Color);
-  --pf-c-nav__link--BackgroundColor: var(--pf-c-nav__tertiary-list__link--BackgroundColor);
-  --pf-c-nav__link--hover--BackgroundColor: var(--pf-c-nav__tertiary-list__link--hover--BackgroundColor);
-  --pf-c-nav__link--focus--BackgroundColor: var(--pf-c-nav__tertiary-list__link--focus--BackgroundColor);
-  --pf-c-nav__link--active--BackgroundColor: var(--pf-c-nav__tertiary-list__link--active--BackgroundColor);
-  --pf-c-nav__link--m-current--BackgroundColor: var(--pf-c-nav__tertiary-list__link--m-current--BackgroundColor);
-  --pf-c-nav__link--before--BorderColor: var(--pf-c-nav__tertiary-list__link--before--BorderColor);
-  --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__tertiary-list__link--before--BorderWidth);
-  --pf-c-nav__scroll-button--before--BorderColor: var(--pf-c-nav__tertiary-list__scroll-button--before--BorderColor);
-  --pf-c-nav__scroll-button--disabled--before--BorderColor: var(--pf-c-nav__tertiary-list__scroll-button--disabled--before--BorderColor);
-
-  .pf-c-nav__link:hover {
-    --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__tertiary-list__link--hover--before--BorderWidth);
-  }
-
-  .pf-c-nav__link:focus {
-    --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__tertiary-list__link--focus--before--BorderWidth);
-  }
-
-  .pf-c-nav__link:active {
-    --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__tertiary-list__link--active--before--BorderWidth);
-  }
-
-  .pf-c-nav__link.pf-m-current {
-    --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__tertiary-list__link--m-current--before--BorderWidth);
-  }
-}
-
 // Subnav
 .pf-c-nav__subnav {
   --pf-c-nav__link--FontSize: var(--pf-c-nav__subnav__link--FontSize);
```

Comment:
in this block can you create a selector for `.pf-c-nav__link` and use SCSS nesting?